File: 0001-two-sum/0001-two-sum.py
class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """
        
        hmap = {}
        for i in range(len(nums)):
            if nums[i] in hmap:
                return [hmap[nums[i]], i]
            hmap[target - nums[i]] = i 
        
    
            

File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target);
};

vector<int> Solution::twoSum(vector<int>& nums, int target){
    int length = nums.size();
    vector<int> array;
    for(int i = 0; i < length; i++){
        for(int j = i; j < length; j++){
                if(i != j){
                    if((nums[i] + nums[j]) == target){
                        array.push_back(i);
                        array.push_back(j);
                        break;
                    }
                }
        }
    }
    return array;
};